sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_AnalyzeWithPrebuiltModel.md

```diff
@@ -0,0 +1,260 @@
+# Analyze a document with a prebuilt model
+
+This sample demonstrates how to extract text and key information from documents with one of the service's prebuilt models, using an invoice as an example. For a list of the types of documents supported by the Form Recognize service's prebuilt models, please check the [Choosing the prebuilt model ID][choosing-the-prebuilt-model-id] section.
+
+To get started you'll need a Cognitive Services resource or a Form Recognizer resource.  See [README][README] for prerequisites and instructions.
+
+## Creating a `DocumentAnalysisClient`
+
+To create a new `DocumentAnalysisClient` you need the endpoint and credentials from your resource. In the sample below you'll use a Form Recognizer API key credential by creating an `AzureKeyCredential` object, that if needed, will allow you to update the API key without creating a new client.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateDocumentAnalysisClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var credential = new AzureKeyCredential(apiKey);
+var client = new DocumentAnalysisClient(new Uri(endpoint), credential);
+```
+
+## Choosing the prebuilt model ID
+
+The model to use for the analyze operation depends on the type of document to be analyzed. These are the IDs of the prebuilt models currently supported by the Form Recognizer service:
+
+- prebuilt-businessCard: extracts text and key information from English business cards. [Supported fields][businessCard_fields].
+- prebuilt-idDocument: extracts text and key information from US driver licenses and international passports. [Supported fields][idDocument_fields].
+- prebuilt-invoice: extracts text, selection marks, tables, key-value pairs, and key information from English invoices. [Supported fields][invoice_fields].
+- prebuilt-receipt: extracts text and key information from English receipts. [Supported fields][receipt_fields].
+
+## Use a prebuilt model to analyze a document from a URI
+
+To analyze a given file at a URI, use the `StartAnalyzeDocumentFromUri` method. The returned value is an `AnalyzeResult` object containing data about the submitted document. Since we're analyzing an English invoice, we'll pass the model ID `prebuilt-invoice` to the method.
```

Comment:
not sure what u mean with `an English invoice`. so if I have an invoice from somewhere else, what do I pass?

Reply:
I used the term `English invoice` because it was [used by the service team here](https://github.com/Azure/azure-rest-api-specs-pr/blob/paulhsu/formrecognizer.v3.0-preview.1/specification/cognitiveservices/data-plane/FormRecognizer/preview/v3.0-preview.1/doc/Concepts.md#models). From my understanding it means that the invoice prebuilt model is not meant to be used for invoices in other languages. Is this a misunderstanding from my side?

Reply:
Maybe not, but I think it is irrelevant and will just cause confusion. maybe just deleting that word will make it better